packages/schematics/src/collection/ngrx/files/__directory__/__fileName__.facade.spec.ts__tmpl__

```diff
@@ -0,0 +1,102 @@
+import { NgModule } from '@angular/core';
+import {TestBed, async, fakeAsync} from '@angular/core/testing';
+
+import {Observable, combineLatest } from 'rxjs';
+import {take} from 'rxjs/operators';
+
+import { EffectsModule } from '@ngrx/effects';
+import { StoreModule, Store } from '@ngrx/store';
+
+import { NxModule } from '@nrwl/nx';
+import { DataPersistence } from '@nrwl/nx';
+
+import { <%= className %>Effects } from './<%= fileName %>.effects';
+import { <%= className %>Facade } from './<%= fileName %>.facade';
+
+import { <%= propertyName %>Query } from './<%= fileName %>.selectors';
+import { Load<%= className %>, <%= className %>Loaded } from './<%= fileName %>.actions';
+import { <%= className %>State, <%= className %>, initialState, <%= propertyName %>Reducer } from './<%= fileName %>.reducer';
+
+interface TestSchema {
+  '<%= propertyName %>' : <%= className %>State
+}
+
+/**
+ * Helper functions to easily check values in observable properties
+ */
+const valueFrom = (source$) => {
+   let actual;
+   source$.pipe(take(1)).subscribe(value => actual = value);
+   return actual;
+};
+
+describe('<%= className %>Facade', () => {
+  let facade: <%= className %>Facade;
+  let store: Store<TestSchema>;
+  let create<%= className %>;
+
+  beforeEach(() => {
+    create<%= className %> = ( id:string, name = '' ): <%= className %> => ({
+       id,
+       name: name || `name-${id}`
+    });
+  });
+
+  describe('used in feature library', () => {
```

Comment:
facaded can be generated in the root of the application. 'used in a feature library' doesn't really work there.